docs/intro/10-development-environment.rst

```diff
@@ -72,6 +72,18 @@ Windows:
 
 .. _yarn-setup:
 
+NPM *
+-------
+
+NPM is the package manager for JavaScript and the world’s largest software registry.
+A ``package-lock.json`` file should be synced with ``yarn.lock`` by using a package.json script:
+    ``"sync-npm": "rm -f package-lock.json && npm shrinkwrap && mv npm-shrinkwrap.json package-lock.json"``
+That ``package-lock.json`` file is need for dependency analysis and auditing security issues.
+
+Starting with version 6 ``npm audit`` will list security vulnerabilities found in the currently installed packages.
```

Comment:
How do we find the responsible package and fix them?

Reply:
Simply running `npm audit` will list security issues and how to fix them if possible.